src/PKSim.Infrastructure/ProjectConverter/v7_3/Converter721To730.cs

```diff
@@ -56,10 +64,10 @@ public class Converter721To730 : IObjectConverter,
 
       public (int convertedToVersion, bool conversionHappened) ConvertXml(XElement element, int originalVersion)
       {
-         //nothing to do here
-         return (ProjectVersions.V7_3_0, false);
+         var (_, converted) = _coreConverter.ConvertXml(element);
```

Comment:
I had forgotten to also implement the xml conversion required to save the old value description into value origin